apps/emqx/src/emqx_persistent_session_ds_stream_scheduler.erl

```diff
@@ -303,6 +357,12 @@ compare_streams(
 is_fully_replayed(Comm1, Comm2, S = #srs{it_end = It}) ->
     It =:= end_of_stream andalso is_fully_acked(Comm1, Comm2, S).
 
+is_fully_acked(_, _, #srs{
+    first_seqno_qos1 = Q1, last_seqno_qos1 = Q1, first_seqno_qos2 = Q2, last_seqno_qos2 = Q2
```

Comment:
Q: a freshly created `#srs{}` will be in this condition; should it be considered fully acked as well?

Reply:
Yes, I don't see why we can't fetch from a freshly discovered stream immediately.